README.md

```diff
@@ -33,6 +33,13 @@ Details can be found on the project website [here](https://eclipse-xpanse.github
 
 Details can be found on the project website [here](https://eclipse-xpanse.github.io/xpanse-website/docs/database).
 
+## Deployment
+
```

Comment:
Also document the step about how to get the yaml file and with which name it must be copied to which location.

Reply:
start terraform-boot to access“ http://localhost:9090/v3/api-docs" get json file and convert it to yaml format

Reply:
json file can also be used directly without converting to yaml.